cli/cmd/root.go

```diff
@@ -36,7 +36,14 @@ func NewCmdRoot() *cobra.Command {
 	logFileName := "cloudquery.log"
 	sentryDsn := sentryDsnDefault
 
-	err := telemetryLevel.Set(getEnvOrDefault("CQ_TELEMETRY_LEVEL", telemetryLevel.Value))
+	// support legacy telemetry environment variable,
+	// but the newer CQ_TELEMETRY_LEVEL environment variable takes precedence
+	defaultTelemetryValue := telemetryLevel.Value
+	legacyTelemetry := os.Getenv("CQ_NO_TELEMETRY")
+	if legacyTelemetry != "" {
+		defaultTelemetryValue = "none"
```

Comment:
[Non blocking] Maybe add a warning message here to switch to the new env?

Reply:
Good idea, added 👍